contrib/pom.xml

```diff
@@ -623,5 +628,11 @@
       <artifactId>super-csv-joda</artifactId>
       <version>2.3.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.gemstone.gemfire</groupId>
```

Comment:
That would be preferred. We would get the dependency from Apache snapshots,
right? In that case, no extra repository declaration should be required.

On Tue, Jan 19, 2016 at 1:50 PM, William Markito <notifications@github.com>
wrote:

> In contrib/pom.xml
> <https://github.com/apache/incubator-apex-malhar/pull/125#discussion_r50181242>
> :
>
> > @@ -623,5 +628,11 @@
> >        <artifactId>super-csv-joda</artifactId>
> >        <version>2.3.1</version>
> >      </dependency>
> > +    <dependency>
> > +      <groupId>com.gemstone.gemfire</groupId>
>
> Would you guys like to point it to Geode instead of GemFire ? For now it
> would need to be pulled from incubator repository and from nightlies but a
> release is just about to start.
>
>       <groupId>org.apache.geode</groupId>
>       <artifactId>gemfire-core</artifactId>
>       <version>${geode.version}</version>
> </dependency>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-apex-malhar/pull/125/files#r50181242>
> .
>